Bottom: 01a94df52b4ff33876c1d7340d1328b024b4f3eb
Top:    710fd4841e48bbaf2cfbe54a84e1e39df125d31c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 01:58:02 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index 316208e..73d9a34 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -1,10 +1,8 @@
-/* $Id: gui.h,v 1.25 2008-04-13 14:15:38 petercjclifton Exp $ */
-
 /*
  *                            COPYRIGHT
  *
  *  PCB, interactive printed circuit board design
- *  Copyright (C) 1994,1995,1996 Thomas Nau
+ *  Copyright (C) 2009 PCB Contributors (See ChangeLog for details).
  *
  *  This program is free software; you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
